pandora-common/src/validation.ts

```diff
@@ -178,4 +178,5 @@ export const SIMPLE_TOKEN_LENGTH = 6;
 export const SimpleTokenSchema = z.string().length(SIMPLE_TOKEN_LENGTH).regex(/^[0-9]+$/);
 export const IsSimpleToken = ZodMatcher(SimpleTokenSchema);
 
+// TODO: set minimum password length to maybe 8 characters?
```

Comment:
Password length cannot be enforced here, server does not have any information about the password length, TODO could be put in limits maybe

or better yet added as validation in `registrationForm.tsx` and `resetPaasswordForm.tsx`

Reply:
I just pushed some code that shows that I can enforce password length here together with some client code in those two files you mentioned.

It worked in my tests. Is it not correct like I did in my latest commit?